pyproject.toml

```diff
@@ -119,13 +119,48 @@ module = [
 ]
 
 # Gradually we want to add more modules to this list, ratcheting up our total
-# coverage. Once a module is here, functions require annotations in order to
-# pass mypy. It would be especially useful to have tests here, because without
-# annotating test functions, we don't have a great way of testing our type
-# annotations — even with just `-> None` is sufficient for mypy to check them.
+# coverage. Once a module is here, functions are checked by mypy regardless of
+# whether they have type annotations. It would be especially useful to have test
+# files listed here, because without them being checked, we don't have a great
+# way of testing our annotations.
 [[tool.mypy.overrides]]
-disallow_untyped_defs = true
```

Comment:
Probably we want to have both checks enabled?

Reply:
This is much stricter, and so can't be rolled out to much of the library at all (and also much less important imo)

Reply:
We could have another category for this, but I don't think it's worthwhile, my original push was always to get more checks running